public/refs/paths/accounts_{account_id}_sites_{site_identifier}_subjects_{subject_number}_study_events_{study_event_identifier}_study_procedures_{study_procedure_identifier}_assets_move.json

```diff
@@ -0,0 +1,118 @@
+{
+  "put": {
+    "summary": "Move assets between study procedures",
+    "description": "Move assets between study procedures",
+    "operationId": "moveAssets",
+    "tags": ["Study Procedure Assets"],
+    "security": [
+      {
+        "userSessionKey": [],
+        "chillipharmSession": []
+      }
+    ],
+    "parameters": [
+      {
+        "name": "account_id",
+        "in": "path",
+        "description": "The account id",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "1"
+        }
+      },
+      {
+        "name": "site_identifier",
+        "in": "path",
+        "description": "The site identifier",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "SITE1"
+        }
+      },
+      {
+        "name": "subject_number",
+        "in": "path",
+        "description": "The subject number",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "12345678"
+        }
+      },
+      {
+        "name": "study_event_identifier",
+        "in": "path",
+        "description": "The study event identifier",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "VISIT1"
+        }
+      },
+      {
+        "name": "study_procedure_identifier",
+        "in": "path",
+        "description": "The study procedure identifier",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "PROC1"
+        }
+      }
+    ],
+    "requestBody": {
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "properties": {
+              "authenticity_token": {
+                "type": "string",
+                "description": "The authenticity token",
+                "example": "valid_token"
+              },
+              "items": {
```

Comment:
its not true, the `AssetsMove` content should be part of the `properties` and not be nested into `items`

Reply:
it means that all the properties from `AssetMove` ref should be put next to `authenticity_token` and **not** inside `items` property

What you have now:

```json
{
  "body": {
    "authenticity_token": "valid_token",
    "items": {
      "assets": [1, 4, 100],
      "subject_number": "E0010023",
      "study_event_identifier": "VISIT1",
      "study_procedure_identifier": "NSTAR",
      "reason_for_moving": "Mistake"
    }
  }
}
```

vs what you should have:

```json
{
  "body": {
    "authenticity_token": "valid_token",
    "assets": [1, 4, 100],
    "subject_number": "E0010023",
    "study_event_identifier": "VISIT1",
    "study_procedure_identifier": "NSTAR",
    "reason_for_moving": "Mistake"
  }
}
```